### DISCUSSION TITLE
How to set max of ss_prog1_power to higher value to match inverter allowed settings

### DISCUSSION THREAD

**Discussion by f1wade** [kellerza/sunsynk#185] [General]
Hi, I have a 5kw inverter, but it allows me to set the power values within each timer section to up to 5500.
but this add on to HA, only sets the max to rated power, which seems to be 5000, even though it can go higher.

I have tried adding the value to the mysensors file, but it doesn't reset them.

Is there a way to make this setting higher?

I can edit it in the HA state in developer tab, but on restart that all resets.

**Comment by kellerza** [comment]
This worked for another user - #181 (use the first line in my comment, max=5500)

**Reply by f1wade** [reply]
I tried that but it didn't update the max. 
I could use a different name and that works.
But I wondered if there was a way to adjust the sensors that are already there?
